### PR TITLE
Update Move Category icons

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Updates them to use modern palettes and a few other tweaks as well

## Images
pokeemerald-0
pokeemerald-1
pokeemerald-2

## **Discord contact info**
kittenchilly

### CODE REVIEW

**Comment by hedara90** [comment]
I don't know if I agree that they DPPt ones look better. Personally I don't like the color contrast on Physical, and Special lost the "special color" in my eyes, and I don't like the that the top/bottom margins are different. I prefer the HGSS ones over DPPt.
But it's matter of taste. Would probably need more voices in any case.

**Comment by Pawkkie** [comment]
> Would probably need more voices in any case.
I also prefer the HGSS icons, especially for the Special icon colour. I agree with hedara. I don't necessarily mean for this agreement to be definitive, but I am certainly willing to add an extra voice.

**Comment by kittenchilly** [comment]
> I don't know if I agree that they DPPt ones look better. Personally I don't like the color contrast on Physical, and Special lost the "special color" in my eyes, and I don't like the that the top/bottom margins are different. I prefer the HGSS ones over DPPt. But it's matter of taste. Would probably need more voices in any case.

Physical wasn't even HGSS, its basically the same as what I changed it to. I figured the special color would the most contentious but I can change that. The top/bottom margins being what they were are unlike every other sprite like it in the game, including the type icons and it made some parts of it invisible if they were displayed on a white or black background

**Comment by kittenchilly** [comment]
Updated with color tweaks based more on gen 5

File: graphics/interface/category_icons.png
Comment (hedara90) [inline graphics/interface/category_icons.png:1]: Top part of status icon could be less redish.
`d0a99c` -> `e0ddc7` to match HGSS color.
Current -> New
20240814_12h08m51s_grim

**Review by hedara90 — DISMISSED** [review]
Could you add a config to keep the old icons?
`#if` around which image gets included and regular `if` for the 129->128 change (if the 129 is required for old icons)

**Comment by kittenchilly** [comment]
> Could you add a config to keep the old icons? `#if` around which image gets included and regular `if` for the 129->128 change (if the 129 is required for old icons)

Well the icons are already custom in the first place, so I'm not sure if this would really make sense. Its like keeping every version of a ds-style pokemon's sprite

**Comment by hedara90** [comment]
> Well the icons are already custom in the first place, so I'm not sure if this would really make sense. Its like keeping every version of a ds-style pokemon's sprite

Oh, right, sorry. I keep forgetting that it wasn't a Gen 3 thing.
Then it's just the color of the top part of the status icon, I think It's a bit too red, but other thoughts are welcome.

File: graphics/interface/category_icons.png
Comment (kittenchilly) [reply graphics/interface/category_icons.png:1]: I changed it, but the HGSS color palette isn't the same as the one that's used, so I just made it more of a light grey than reddish

**Review by hedara90 — APPROVED** [review]

**Pull request merged**